### ISSUE TITLE
 Get data for all 2D material data

### ISSUE THREAD

**Issue by Xiaoshouzhen** [materialsproject/api#763]
How can I get all the contents of the 2D material data?
The old version of the API can be easily obtained, and I can't find a quick way to obtain it in the new version.

**Comment by munrojm** [comment]
Unfortunately, it is not quite as easy as the legacy API at the moment. This is something I can add, and will make it a priority for the next release.

In the mean time, you could pull all of the `condensed_structure` data from the `robocrys` endpoint and filter locally.

**Comment by Xiaoshouzhen** [comment]
> yes sir i am doing this and this endpoint has key value like dimension:integer but the filtering process is unbearable. Looking forward to your contribution to the next version with a faster interface. Thank you from the bottom of my heart and wish you success in your work.

**Comment by Xiaoshouzhen** [comment]
> 不幸的是，目前它不像旧版 API 那样简单。这是我可以添加的内容，并将使其成为下一个版本的优先事项。
> 
> 同时，您可以`condensed_structure`从`robocrys`端点提取所有数据并在本地进行过滤。

yes sir i am doing this and this endpoint has key value like dimension:integer but the filtering process is unbearable. Looking forward to your contribution to the next version with a faster interface. Thank you from the bottom of my heart and wish you success in your work.
